### ISSUE TITLE
Game locks at strategy map.

### ISSUE THREAD

**Issue by Fallguy55** [long-war-2/lwotc#516]
[save_Beta (Test).zip](https://github.com/long-war-2/lwotc/files/3612383/save_Beta.Test.zip)

See attached save file. The issue can be easily reproduced by finishing making contact with new india.

Not really sure, but it says that we've made contact with the blacksite region when India doesn't connect to it, then it zooms in on a chosen's region, tells me that I got a regional bonus, and then stays there. I'm then unable to exit out of that area, but time passes on the map, regardless.

I'm using the following mods:
Improved Camera Updated WotC
Stop wasting my time - wotc
wotc: robojumper's squad select
[wotc] revert overwatch rules change
better second wave mod support
[wotc] additional icons
[wotc + classic] TacticalUI KillCounter
[wotc] Mod Config Menu
[Wotc] Stats on new promotion Screen
[More Environmental Lighting Maps
WotC Male Hair Pack
Wotc Female Hair Pack
Yet another F1
Darker Nights
Blackmarket usage
Payday 2: Masks [Wotc]
Dirty Troopers
Character  Paint Pack
Vindictus Female Hairstyles
Plasman Elite Tattoos
More armor colors
Add Part Names
More Hair colors - wotc compatible
Jsleezys Repurposed gear
stenchfury modular armors
custom bandanas - armour colours/pattern
stenchfury modular helmets
[lw2 cmp] xcom 2 international voices pack
evac all - wotc
[wotc] gotcha again
Free the Hood

**Comment by pledbrook** [comment]
This appears to be stuck on revealing a new Chosen after contacting the region. Not sure the bug with the Blacksite narrative is related. Have you encountered the Chosen yet? Did you defeat one of them? I was expecting to see at least one Chosen on the map since you'd completed the first Liberation Network Tower.

**Comment by Fallguy55** [comment]
At this point I had not encountered any of the chosen yet. I have played further, and after discovering the assassin, the problem persisted (because the assassin is located in mid Asia. 

The problem fixed itself after I discovered the hunter in another mission (which is the chosen that the map locks onto in this bug), but it still has Bradford mention the blacksite. 


Another bug in this save (and my whole game) is that there is no way for me to bring the skirmishers into this game. I enabled Lost and Abandoned at the start, but the lost and abandoned mission never happened, and I have no ability to contact or acquire skirmishers. I've built the ring at this point, still nothing.
